Salt Client registration from Command Line (#765)
uyuni-project/uyuni-docs: modules/client-configuration/pages/registration-cli.adoc

// I'd say "no", according to the outcome of
// https://github.com/SUSE/spacewalk/issues/9333 KE 2019-12-17

In most cases, Salt clients are registered accurately with the default bootstrap methods.
However, you can use Salt to register the client to the {productname} Server manually by editing the Salt minion file on the client, and providing the fully qualified domain name (FQDN) of the server.

> This method operates using ports 4505 and 4506 inbound to the server.

Reviewer: ```suggestion
This method uses ports 4505 and 4506 inbound to the server.
```

For more information, see xref:registration-bootstrap.adoc[].
====

> This procedure requires that you have installed the Salt package on the Salt client before registration ([package]``salt-minion``).

Reviewer: ```suggestion
This procedure requires that you have installed the [package]``salt-minion`` package on the Salt client before registration.
```

----
/etc/salt/minion.d/NAME.conf
----

> . Add or edit the FQDN of the {productname} Server or of the Proxy as the `master`, and the activation key, to the client configuration file:

Reviewer: ```suggestion
. Add or edit the FQDN of the {productname} Server or of the Proxy as the `master`, and add the activation key to the client configuration file:
```

??
